{"name": "Отображается сообщение об ошибке после запроса без id", "status": "passed", "start": 1734079799713, "stop": 1734079799790, "uuid": "5de9e48d-30de-4d6f-ac30-c51805a1ded3", "historyId": "7dd27198fcccd4dcf275c08d7c1560e3", "testCaseId": "7dd27198fcccd4dcf275c08d7c1560e3", "fullName": "tests.test_get_order_using_his_number.TestGetOrder#test_error_message_after_request_without_track_id", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_order_using_his_number"}, {"name": "subSuite", "value": "TestGetOrder"}, {"name": "host", "value": "WIN-3SKQPE0FURM"}, {"name": "thread", "value": "256-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_order_using_his_number"}], "links": [{"type": "link", "url": "https://qa-scooter.praktikum-services.ru", "name": "https://qa-scooter.praktikum-services.ru"}]}